schematics/utils/devkit-utils/component.ts

```diff
@@ -16,7 +16,6 @@ import {
 } from '@angular-devkit/schematics';
 import 'rxjs/add/operator/merge';
 import * as ts from 'typescript';
-import * as stringUtils from '@schematics/angular/strings';
 import { addDeclarationToModule, addExportToModule } from './ast-utils';
```

Comment:
Not something that was added in this PR, but the spaces between the curly braces are inconsistent with the rest of the project.

Reply:
Done